src/DIELECTRIC/fix_polarize_bem_gmres.cpp

```diff
@@ -276,6 +281,15 @@ void FixPolarizeBEMGMRES::setup(int /*vflag*/)
       error->all(FLERR, "No Kspace style available for fix polarize/bem/gmres");
   }
 
+  // NOTE: qqrd2e = e^2/(4 pi e0) in kcal/mol
+  // converting epsilon0 [efield] from (F/m)(kcal/mol/e/A) into e/A^2
+  // epsilon0e2q = epsilon0 * (4184/6.023e+23) * (1/1.6e-19)^2 / (1e+10) / 332.06
+  //             = 0.000240258
+
+  epsilon0e2q = 1.0;
+  if (strcmp(update->unit_style,"real") == 0)
+    epsilon0e2q = 0.0795776 / force->qqrd2e;
```

Comment:
Can we compute this constant with larger precision?
Should this perhaps be moved to conversion constants in the Force class?
Do I understand this correctly that this fix will work properly only with "lj" and "real" units?
I am adding a check and error messages for now.

Reply:
@akohlmey Thanks for pointing out the units dependent factor and for adding the check and error message. In commit 7c64ff6, I have improved the precision of the constant epsilson0e2q. I have also computed this constant for "metal", "si" and "nano". The rest may come later, but these are the commonly used units relevant to these fixes in my opinion. The error message is updated to reflect the supported units. The doc page of the two fix polarize fixes is also updated to mention this restriction.

I can clean up the comments that explain how I calculated this factor for units other than lj, if necessary. For the moment, I only see this conversion factor present only in these two fixes, so no need to move it to the Force class.